contracts/NFTOpt.sol

```diff
@@ -427,4 +328,32 @@ contract NFTOpt {
 
         emit Exercised(_optionId);
     }
-}
+
+    /// @dev -- CUSTOM ERRORS -------------------------

```

Comment:
why don't we define these above, before using them?

Reply:
for readability :)